src/en/config-LXD.md

```diff
@@ -0,0 +1,204 @@
+image's filename is its *full fingerprint*. The fingerprint shown in an image
+list is its *partial fingerprint* (the beginning portion of the cached image's
+filename). Either type of fingerprint can be used to refer to images.
+
+
+## Import an image into the local remote
+
+Import a 64bit Trusty image, tag it to be synced, and create the alias:
+
+```bash
+lxd-images import ubuntu trusty amd64 --sync --alias ubuntu-trusty
+```
+
+This sort of invocation will pull official Ubuntu cloud images from
+http://cloud-images.ubuntu.com.
+
+Once any given image is imported it is cached at
+`/var/lib/lxd/images` and shows up with `lxc image list`:
+
+![lxc image list after importing](./media/image_list-imported_image-reduced70.png)
+
+To sync, run:
+
+```bash
+lxd-images sync
+```
+
+By default, the above is managed via cron (`/etc/cron.hourly/lxd`).
+
+
+# LXD test
+
+Although the objective here is to have Juju launch containers as it needs them,
+as a test for your LXD installation, you should manually launch one now (here
+called 'ubuntu-trusty-64-test').
+
+We refer to the alias created during importation.
+
+```bash
+lxc launch ubuntu-trusty ubuntu-trusty-64-test
+```
+
+List all containers and then remove the test container:
+
+```bash
+lxc list
+lxc delete ubuntu-trusty-64-test
+```
+
+**Note:** Do not confuse command `lxc` with the binary shipped with traditional
+LXC. All the latter's binaries are of the form `lxc-<subcommand>`. The `lxc`
+binary actually comes from the `lxd-client` package.
+
+
+# Configure and bootstrap
+
+If this is a new Juju install then you do not yet have a
+`~/.juju/environments.yaml` file. Create one with
+
+```bash
+juju generate-config
+```
+
+If it does exist (but it was created with an older version of Juju), first move
+it out of the way (back it up) and *then* generate a new one. Alternatively,
+you can output a generic file to screen (STDOUT) and paste the lxd parts into
+your existing file:
+
+```bash
+juju generate-config --show
+```
+
+The file should now contain a section for the LXD local provider. The minimum
+active lines for this section are:
+
+```yaml
+   lxd:
+        type: lxd
+```
+
+For the alpha2 version of Juju (will change), you must specify the
+'--upload-tools' flag when bootstrapping an environment that will use Trusty
+images. This is because, currently, most of Juju's charms are for Trusty, and
+the agent-tools for Trusty don't yet have LXD support compiled in.
+
+Switch to the LXD environment and bootstrap it:
+
+```bash
+juju switch lxd
+juju bootstrap --upload-tools
+```
+
+
+# Logs
+
+LXD itself logs to `/var/log/lxd/lxd.log` and Juju machines created via the
+LXD local provider log to `/var/log/lxd/juju-{uuid}-machine-{#}`. However,
+the standard way to view logs is with the `juju debug-log` command. See
+[Viewing logs](./troubleshooting-logs.html) for more details.
+
+
+# Other useful commands
+
+There is a cornucopia of commands available. Some common ones not yet covered
```

Comment:
While "cornucopia" is correctly used here it stuck out when I read it.  Did "plethora", "abundance", "multiple", or "many" not work in this context?